### DISCUSSION TITLE
想聽什麼音樂先「聽聽自己心跳聲」

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#532] [科技新知]
蘋果在週一時收購了一家名為 AI Music 的新創公司，這家公司最大的特色在於號稱可利用人工智慧技術來自動生成適合每位用戶的音樂。此外，該公司的 AI 技術甚至可以根據每位使用者的心跳，來創造出當下適合的音樂；舉例來說，當你在自我訓練時，AI Music 可會根據你當下的心跳，創作出更慷慨激昂的樂曲來激勵你繼續鍛鍊。
https://finance.technews.tw/2022/02/08/apple-ai-music/

**Comment by ghost** [comment]
原來如此，但其實我更想聽到別人的心跳聲>///<

**Comment by Yuhsin000** [comment]
在做事情之前聽讓自己開心的音樂，可以增加做事的能力 (真實研究，原本被誤當成是莫札特的功勞)，有了這個技術可能可以找到更適合的音樂，提升更多喔~~

**Comment by Sazmafuji** [comment]
希望這技術可以變成熟，讓我們能夠聽想聽的音樂

**Comment by leon10456** [comment]
如果能加上聽好友的心跳就更好了(///)

**Comment by ThomasMeng0515** [comment]
不知道他們是如何偵測心跳的，是不是用戶一定要戴上智慧型手表才能偵測心跳並且給出適合的音樂，如果沒有配戴智慧型穿戴裝置，要偵測心跳感覺就有點困難。

**Comment by 11timyyy** [comment]
感覺很適合在運動時或讀書時使用

**Comment by hapiwangy** [comment]
技術的出發點具有獨特性，但假使能夠克服接收心跳的困難的話（不僅限於使用特定穿戴式裝置）就可以方便許多。

**Comment by dayu1004** [comment]
不知道配合心跳聲的音樂會不會提告做事情的效率 或是睡眠品質更好哈哈

**Comment by lx3211** [comment]
如何準確接收心跳應該是他們主要需要克服的課題之一

**Comment by HuanYuuuuuu** [comment]
感覺在讀書時很適合用到這項功能，不然常常都不知道要挑什麼音樂來聽....

**Comment by Rosancl** [comment]
可以根據當下心跳（行為）來推薦音樂是很棒的功能，希望偵測心跳的方式是簡易的，否則應該沒多少人會願意為了讓系統推薦音樂而特地穿戴其他裝置。

**Comment by yucheng1028** [comment]
一顆心撲通撲通的跳

**Comment by yenwei12138** [comment]
原來~ 是心動的聲音~

**Comment by stan890425** [comment]
心跳的聲音很規律會讓聽的人安心

**Comment by Jimmy911218** [comment]
用心跳來演算要撥哪手歌確實很創新，但不一定心跳聲能符合當時想聽的音樂阿

**Comment by minniewei** [comment]
這個東西真是太創新了，但音樂跟心跳有關倒是我完全沒有想到的

**Comment by liangyinglly** [comment]
我覺得其實還需考慮心情這個不定因子

**Comment by HaoFish123** [comment]
感謝分享，不知道這項技術會不會因為心臟相關疾病而受到影響，畢竟每個人在相同情緒下的心跳也依然會不同。

**Comment by treeann** [comment]
原來如此 音樂就是這麼樸實無華且枯燥

**Comment by HsuChen88** [comment]
有心率不整的人會聽到什麼音樂呢？

**Comment by 110409538** [comment]
很想知道自己的心跳聲會是怎樣的歌單.

**Comment by yi-1111** [comment]
太創新了，感謝分享

**Comment by jcpan821** [comment]
讓我想到獵人

**Comment by alexthebigpp** [comment]
這樣很方便哀，像在運動時就可以讓裝置自動幫你挑選適合的音樂

**Comment by bloodtrail0511** [comment]
好酷的感覺

**Comment by etin2468** [comment]
讀書時很適合用到這項功能，雖然我還是希望以純音樂為主

**Comment by Hchejxnc** [comment]
這個技術真的很有趣，想試試看在各種不同的情況下，它會帶給我什麼樣的音樂。

**Comment by Tuoresa** [comment]
感覺非常有趣！！好想試用看看！
有時候想聽音樂總是找很久還找不到符合當下感覺得歌曲
但也說不上來想聽什麼樣的歌

很期待真正使用到時的感受
或許能解決好想聽音樂卻總是頻率不對的問題哈哈

**Comment by bonny123865** [comment]
感覺這個技術很不錯，可以依照使用者的感受做變更歌曲!
希望未來技術更成熟的時候還能選擇不同的情境!

**Comment by Valentinaa0214** [comment]
超酷的，想不到要聽甚麼音樂就聽聽自己的心跳聲❤️

**Comment by BlahFish1006** [comment]
我覺得也許可以多加一個「預設想達到的狀態的心跳聲」會不會更好呢？
因為即將開始做某件事，但還未做的時候，當下心跳聲應該不會是我們想達到的狀態，
比如我們原本很亢奮，心跳是快速的，但想要靜下心來念書就會需要比較舒緩的音樂，讓心跳得以放慢，
也許這樣能滿足到更多種不同面向的需求~

**Comment by dabbie109303501** [comment]
音樂能影響聽者的情緒 而心跳是反應情緒的一種方式
上學期修過普通心理學 有讀過老師提供的一篇跟音樂和情緒有關的文章
有興趣的人可以看看
[音樂與情緒.pdf](https://github.com/NCU-GS4719-Python/Python-Community/files/10366114/default.pdf)

**Comment by wenchi9023** [comment]
像我自己在跑步的時候很喜歡聽節拍重的音樂，讓自己運動起來更有勁
如果AI能根據我的心跳快慢調整音樂，我想會讓我更又動力運動!
蘋果真的很會把產品提升到另一個更人性化的層次
期待這個功能趕快在蘋果產品上出現~

**Comment by harrylee1971** [comment]
運動型的AI可以找到讓人訓練更開心快樂的方式，因為音樂是有激勵人心的作用的阿

**Comment by a45320614** [comment]
除了在運動時使用很實用外，我認為也很適合在睡前撥放相對應的助眠音樂，有了這個技術，也不再有在推薦列表找來找去就是找不到符合自己心情的那一首歌的煩惱了!

**Comment by HsuChen88** [comment]
對於患有心律不整的我來說，實在很想測試一下這個工具，因為我的心跳就是不規則的，想看看他是否會做出掰咖的音樂。

**Comment by 110302550** [comment]
超酷的ㄟ，感覺有個人化的感覺，專屬於於自己的音樂，我覺得很有創意，會讓我想買單
